modules/mono/editor/GodotTools/GodotTools.IdeMessaging/GodotTools.IdeMessaging.csproj

```diff
@@ -1,10 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProjectGuid>{92600954-25F0-4291-8E11-1FEE9FC4BE20}</ProjectGuid>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <LangVersion>7.2</LangVersion>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
I think the JetBrains team is using this library in their [Rider extension](https://plugins.jetbrains.com/plugin/13882-godot-support), so we need to make sure this won't break them (also the other IDE plugins but those are less maintained).

For example, our VSCode extension still uses .NET Framework:

https://github.com/godotengine/godot-csharp-vscode/blob/5cb26cc7fa4059d2aaacc5fdf46a7008d6c40951/GodotDebugSession/GodotDebugSession.csproj#L12

Reply:
Looking at the VSCode Extension, it's calling to version `1.1.0`, which is already bumped up to `1.1.1` in the main repo & `1.1.2` in this PR. I'm not entirely sure where the extension is even pulling the package from, but so long as the version number is incremented I would imagine that this wouldn't cause any overlapping issues

Reply:
Yes, the extension could just not upgrade the dependency but ideally they'd stay up to date so they can get bug fixes and new features. The VSCode extension specifically is a bit under-maintained at the moment, so I care a bit more about the needs of the JetBrains Rider extension.

If changing the TFM makes upgrading harder for consumers then that's not a good situation to be in. So, if a lot of our consumers are still on .NET Framework, I think changing the TFM would not be worth it unless it brings significant benefits which I don't think it does here.

Reply:
Yeah, that's fair. Out of all the projects in this PR, this would be the most likely candidate to remain as `netstandard2.0`. I'd be fine with that, as it'd still mean no more `net472` projects

Reply:
Reverted this project entirely. This will be better suited for a PR with IDEs in mind explicitly